frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.test.tsx

```diff
@@ -23,21 +23,6 @@ const form = (
   </Formik>
 );
 
-describe('field definition conditional render', () => {
```

Comment:
Field description no longer used by this component. Replaced by conditional text for Surplus Activce/Encumbered